sdk/eventhub/event-processor-host/src/blobService.ts

```diff
@@ -144,7 +144,7 @@ export class BlobService {
             containerName,
             result.exists
           );
-          resolve(result.exists);
+          resolve(!!result.exists);
```

Comment:
I like !! idiomatically, but this might be a little more clear for non-experts:

```suggestion
          resolve(Boolean(result.exists));
```